### ISSUE TITLE
Persistent diff when using `manage_default_network_acl` with subnets created outside config

### ISSUE THREAD

**Issue by lorengordon** [terraform-aws-modules/terraform-aws-vpc#736]
## Description

When using `manage_default_network_acl = true`, creating any subnets outside of the terraform config results in a persistent diff.

## Versions

 - Terraform: 1.1.2

 - Provider(s): 3.73.0

 - Module: master branch

## Reproduction

1. cd examples/network-acls
2. run terraform apply
3. manually create a new subnet in the vpc
4. run terraform apply

## Expected behavior
Expected a clean terraform plan.

## Actual behavior

Terraform displays a persistent diff that cannot be reconciled.

```
Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # module.vpc.aws_default_network_acl.this[0] will be updated in-place
  ~ resource "aws_default_network_acl" "this" {
        id                     = "acl-0ec4aa76293d0d15f"
      ~ subnet_ids             = [
          - "subnet-075246afce2e34609",
            # (3 unchanged elements hidden)
        ]
        tags                   = {
            "Environment" = "dev"
            "Name"        = "network-acls-example"
            "Owner"       = "user"
        }
        # (5 unchanged attributes hidden)


        # (4 unchanged blocks hidden)
    }

Plan: 0 to add, 1 to change, 0 to destroy.
```

## Additional context

There is a prior issue reporting the same diff, but when _only_ using the module to manage subnets, https://github.com/terraform-aws-modules/terraform-aws-vpc/issues/529.

That issue was resolved by adding all the subnets to the default nacl, https://github.com/terraform-aws-modules/terraform-aws-vpc/pull/530.

However, that does not work when the subnet is created outside this module.

**Comment by lorengordon** [comment]
I'll have a pull request shortly, implementing the lifecycle ignore_changes option...

**Comment by antonbabenko** [comment]
This issue has been resolved in version 3.11.5 :tada:
